src/KM_Controls.pas

```diff
@@ -905,6 +906,7 @@   TKMColumnBox = class(TKMControl)
     procedure SetColumns(aHeaderFont: TKMFont; aCaptions: array of string; aOffsets: array of Word);
     procedure AddItem(aItem: TKMListRow);
     procedure Clear;
+    procedure RevertLastItemIndexChange;

```

Comment:
Yes, its kind of a "trick" to solve my issues. 
Why did I add it: ItemIndex of list is updated on every click on it - so when I click All for AI this DropColumn will be updated to ItemIndex = 2, even if should still be 0, because there this ColumnBox is the last (f.e.) and there are not enought slots to set it to AI too.

The same thing could be done if we save this concrete ItemIndex not in the control but in Lobby class in some array or smth like that.

Also there are some issues, because of double usage of ColumnBox as separate control and as DropDown item - behaviour is similar but have some differences. May be its good idea to split it into 2 controls in the future.